osm2gtfs/creators/ci_abidjan/config.json

```diff
@@ -0,0 +1,35 @@
+{
+    "query": {
+        "bbox": {
+            "n": "5.6367",
+            "s": "5.2121", 
+            "e": "-3.7154", 
+            "w": "-4.2973" 
+        },
+        "tags": {
+            "route": ["bus", "ferry"]
+        }
+    },
+    "stops": {
+        "name_without": "Add a name to the stop with JungleBus app",
+        "name_auto": "yes"
```

Comment:
I think we can do without this option

Reply:
name_without or name_auto or both?

Reply:
DONE : name_without modified set to 'Arrêt sans nom' and name_auto set to no.